src/departure-list/section-items/line.tsx

```diff
@@ -138,11 +131,14 @@ export default function LineItem({
             {title}
           </ThemeText>
         </TouchableOpacity>
-        <ToggleFavoriteDepartureButton
-          line={group.lineInfo}
-          stop={stop}
-          quay={quay}
-          mode={mode}
+        <ToggleFavouriteDeparture
+          existingFavorite={existingFavorite}
+          onMarkFavourite={onMarkFavourite}
+          toggleFavouriteAccessibilityLabel={
+            mode === 'departures'
+              ? toggleFavouriteAccessibilityLabel
+              : undefined
```

Comment:
Is this check needed, or could we just always use `toggleFavouriteAccessibilityLabel`?